code/modules/food_and_drinks/food/snacks_pie.dm

```diff
@@ -246,3 +246,23 @@
 	bonus_reagents = list(/datum/reagent/consumable/nutriment = 4, /datum/reagent/consumable/nutriment/vitamin = 6)
 	tastes = list("mint" = 1, "pie" = 1)
 	foodtype = GRAIN | FRUIT | SUGAR
+
+/obj/item/reagent_containers/food/snacks/pie/baklava
+	name = "baklava"
+	desc = "A delightful healthy snake made of nut layers with thin bread."
```

Comment:
Snake? Should this be snack (same for slice)

Reply:
*Yep*